test/support/factories.cr

```diff
@@ -44,7 +44,7 @@ module Shards
     def create_git_commit(project, message = "new commit")
       Dir.cd(git_path(project)) do
         run "git add ."
-        run "git commit --allow-empty -m '#{message}'"
+        run "git commit --no-gpg-sign --allow-empty -m '#{message}'"
```

Comment:
Maybe useful, but unrelated to this pull request.

Reply:
Well, it was required to run the test suite without being asked about signing any random commits.